packages/cfd/src/Utils/Validator/validator.ts

```diff
@@ -1,9 +1,54 @@
 import { template } from '_common/utility';
 import { getPreBuildDVRs } from '@deriv/shared';
 import Error from './errors';
+import { TCoreStores } from '@deriv/stores/types';
+
+interface Rule {
```

Comment:
What is the reason of using interface and types ?
I'd rather use interfaces for classes but let's stick to types since we only use types all over our project.

Reply:
Okay sure will convert to type